test/performance/throughput/ThroughputSubscriber.cpp

```diff
@@ -126,6 +127,7 @@ void ThroughputSubscriber::DataReaderListener::on_data_available(
             lost_samples_ += last_seq_num - last_seq_num_ - size;
         }
         last_seq_num_ = last_seq_num;
+        received_samples_ += 1;
```

Comment:
```suggestion
        received_samples_ += size;
```

Reply:
Fixed. 
I applied another change, because there is a possibility that a received element is not valid, and that counts as a loss, so `size` variable should not be used here.